vars/newmanExecute.groovy

```diff
@@ -128,8 +164,33 @@ void call(Map parameters = [:]) {
                         config: config.plus([newmanCollection: collection]),
                         collectionDisplayName: collectionDisplayName
                     ]).toString()
+                def command_secrets
+                if(config.cfAppsWithSecrets){
+                    CfUtils cfUtils = new CfUtils();
```

Comment:
You don't define a class in `CfUtils.groovy`... I've found [this](https://www.logicbig.com/tutorials/misc/groovy/class-vs-script-file.html) only, apparently CfUtils is a Script?

Reply:
Yes indeed @stippi2 . I found quite some implementation in the library in a similar mode; and hence I followed the same. In fact my first implementation had a class and there was not much advantage i could see with that approach in this specific context (perhaps some less params for some functions and that's it - couldn't really find a bigger purpose that we normally see in class).

Having said this; i really would like to hear your comments and see whether I have overlooked some aspects.
Thanks!

Reply:
No, fine, I haven't done any programming in Groovy before January. I am still learning. :-)

Reply:
Thanks @stippi2 . Now i renamed the variables as well and pushed so that we have consistent naming followed.